### PR TITLE
Emoji svg

### PR DESCRIPTION
**Pull request by dougfelt**
Updates the svg cleaner tool, adds most missing SVG (excepting flags), and applies the cleaner to the svg.

### CODE REVIEW

**Comment by asturur** [comment]
this is fantastic.
If i can add a suggestion, i see you passed SVGO over the svgs, adding the width and height to the main tag that helps a lot.
The point is that for 128px people already have pngs, faster to open and parse. I would use a bigger width/height value ( 512px up ) since when using svg as normal images shrinking down is always possible while enlarging requires tweaking the svg header ( if opened as a raster )

**Comment by dougfelt** [comment]
Hmmm, that's an option.  Originally the goal was to build a version of the emoji font using embedded svg instead of png, but that's been on hold for a few years.  And we could also tweak the extent during the build process were we to revive that.  File a feature request at the upstream googlei18n/noto-emoji repo.

**Comment by asturur** [comment]
to be honest the font is cool, but for the web svgs are the best thing.
Every OS has a different emoticon set, we are using this SVG set as a glyph substitution in order to give the same experience to all users.

And works pretty good.

We manually tweak the svgs to be 512px so they do not look pixellated on big font sizes.

File: svg_cleaner.py [hunk 1]
```diff
@@ -157,8 +172,19 @@ def _clean_elem(self, node):
...
+            del n.attrs['i:extraneous']
           if not n.attrs and len(n.contents) == 1:
             n = n.contents[0]
+        elif n.name == 'i:pgf' or n.name == 'foreignObject':
+          continue
+        elif n.name =='switch' and len(n.contents) == 1:
+          n = n.contents[0]
+        elif n.name == 'style':
+          # some emacsen don't default 'style' properly, so leave this in.
+          if False and n.attrs.get('type') == 'text/css':
```
Comment (jungshik) [inline svg_cleaner.py]: So, this line (always false if-clause) is kinda  reminder to delete it later when things change, I guess.

Comment (dougfelt) [inline svg_cleaner.py]: Oh, hmmm, yeah.  Probably should have commented the lines out instead.

**Pull request merged**